### DISCUSSION TITLE
Støtte for Aksel DS linting

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1899] [Komponenter]
Vi vurderer støtte for linting av tokens i Aksel. Dette vil bli implementert med `Eslint` eller `Stylelint`. 
Har du innspill til noe som vil gjøre utvikleropplevelsen god?

**Comment by KenAJoh** [comment]
Features/idèer:
- Warning ved overskriving av `navds`/`navdsi`-prefix
- Error ved bruk av ikke-eksisterende token `--a-` eller `--ac-``
- Error ved overskriving av "intern" token `--__ac`
- ? Warning ved bruk av deprecated token (finnes ikke lengre, men eksisterte før) ?
- Støtte for dette for både CSS, Sass og less(optional om vi trenger less-støtte). Kanskje CSS-in-JS også?

**Comment by KenAJoh** [comment]
https://aksel.nav.no/grunnleggende/kode/stylelint
